_docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -20,13 +20,19 @@ curl localhost:9200/_cluster/settings?pretty
 If necessary, reenable allocation:
 
 ```sh
-curl -X PUT localhost:9200/_cluster/settings -d '{"transient":{"cluster.routing.allocation.enable":"all"}}'
+curl -X PUT localhost:9200/_cluster/settings -H "Content-Type: application/json" -d '{"transient":{"cluster.routing.allocation.enable":"all"}}'
 ```
 
 If reenabling allocation doesn't restore the cluster, [manually reassigning unallocated shards may be necessary]((#force-reallocation %})
 
 ### Check cluster health
-Log into any `elasticsearch_*` node and [check the cluster health]((#check-cluster-health">}}). If the cluster state is not green or has a high number of pending tasks then ElasticSearch cannot accept data from the ingestors.  Resolve the issues with the ElasticSearch cluster and once its state is green, the ingestor queues should be begin to drain.
+Log into any `elasticsearch_*` node and check the cluster health:
```

Comment:
This is likely a basic question, but do we have any instructions/documentation on how to login to the Elasticsearch nodes?  Does it follow an existing pattern we have established elsewhere (such as just logging into VMs via BOSH), or is it unique to anything else?

Reply:
It is the same process as any other bosh deployed VM. It might be worth changing this to `bosh ssh elasticsearch_*` to be explicit.

Reply:
Sounds good, thank you!